### ISSUE TITLE
Review of editorial edits

### ISSUE THREAD

**Issue by fantasai** [w3c/AB-public#167]
I looked through the commit logs since January, and I have a few comments on the changes which are marked as editorial:

```
-	This document is an articulation
-	of W3C’s mission, its values, and its organizational principles;
-	in other words, our vision for W3C as an organization
+	This document articulates
+	W3C’s mission, its values, its organizational principles, 
+	and our vision for W3C as an organization
```

Switching from "in other words" to "and" doesn't actually work here: it's not 4 items we're listing here, it's 3 items that combine into creating the 4th.

```
- * Be timeless enough that it does not need frequent revision.
+ * Be timeless enough to guide W3C yet flexible enough to evolve when needed.
```

I don't think I agree with this edit. The resulting text is a non-sequitur... Timelessness isn't a requirement to guidance, and doesn't create flexibility to evolve either... It's nice to be timeless, and it's nice to be flexible to evolve, but this new sentence as a whole doesn't make sense.

> https://github.com/w3c/AB-public/commit/36bfccf246cb0c1603a7edc896d4331fe1e698fb shifting the mission of W3C into the Vision for W3C section, and the founding of W3C into the middle of the Introduction

This edit breaks the flow between the paragraph about the creation of the Web and the consequences of it with an aside about the founding of W3C. I think I understand what you were trying to do: to combine the historical paragraphs into the Intro, and combine the paragraphs about what W3C should be into the "Vision"... but the result isn't quite working, I think?

It also results in repeating the "equitable, informed, and interconnected society" bit twice in the Vision section, so that probably needs some clean-up. (Maybe remove the last sentence?)

**Comment by cwilso** [comment]
We think we've addressed the issues here.  Do you agree?

**Comment by mgendler** [comment]
@fantasai Have your issues been addressed here?

**Comment by fantasai** [comment]
Yes, I believe they have been, and we just forgot to close the issue!
